Lab#2.ipynb

```diff
@@ -0,0 +1,722 @@
Nuxq4MqlMwa/wDkXW75SPsPsOnQECaaUjI4ur5wcsXC6r5h3Lht4QMkwexqOUFFcSEHe6PnRLqwsYdPuRrq6e0/yc85lkqR+60j6maQPSho9v6Vcn3YfamXaKOnu6rFoeilHOrrYVucD885lsmQDxPeBPwbekHS7pBMjrJM7BrsbjlA5SmZ49fCt7J3LDkkFFDN70sw+DpwGbAeekPTfkj4lKT/KCrrkmRl7DrWOmjUoPeZWjKMwL8e3sncuwyXdhSVpEnANcB2wDvgOQYB5IpKauSE7dLiDIx1do2Ifr0R5uTmcNK3Et2BxLsMlO4byMPAcMBa41MwuM7MHzOzPgaydojva7G0cHdvW9yXYgqWRbh+Ydy5jJdtC+aGZLTCzb/RsviipEMDMlkZWOzckPQFlamlhmmvyTouml9LU1snOg4fTXRXnXESSDShf7+Pc71NZEXfs9ocBZfL40dXlBT4w71w2GHClvKSpBNvDj5G0hD/s9ltC0P3lRpF9jW0ATC4ZfS2U+VOKyc8VG3Y38MFTpqW7Os65CAy29cpFBAPxVcC3Es43Af83ojq5Ydrb2MrEcQUU5uWmuyrvUJiXy/FTxvtML+cy2IABxczuA+6T9FEz+9kI1ckN0/7G1lGz5UpfFk0v5YnN+zAzggdzOucyyWBdXp8wsx8DsyV9offnZvatPi5zabK3sZUpo7C7q8eiyhIeWLOTPQ2j44mSzrnUGmxQflz4tRgY38fLjSL7GtuYMgoH5HssPDow791ezmWiwbq8/jn8+tcjUx03XJ1d3dQ1tzFllC1qTHTS1BJyBBt2N3Lhwqnpro5zLsUG6/L67kCfm9lnUlsdN1y1zW2YMaq7vMYU5DJvcrE/Y965DDXYLK+1I1ILd8x6pgyP5i4vCAbmf/dmXbqr4ZyLQDKzvFwM7Du6Sn50B5SFlaU8vG4X+5taR+UCTOfc8A3W5fVtM/ucpJ8D79iEycwui6xmbkh6AspoXNSYqOcZ8xt3NzL5BA8ozmWSwbq87g+/fjPqirhjs6+xldwcUT5udAeUozO9aho454TJaa6Ncy6VBuvyWht+fVZSAXAiQUtli5m1j0D9XJL2NbYxeXwhOTmje8FgcWEex5WP863snctAyW5f/0HgTeC7wD8C1ZIujrJibmj2NbYyeRSvkk+0sLLUN4l0LgMlu9vw3wPnmNkHzOz9wDnAndFVyw3VvsZWpo7y8ZMei6aXsOvQEQ62eCPXuUySbEDZb2bVCe+3AvsjqI8bpn2NbaN6H69EPVvZb9ztrRTnMsmAAUXSRyR9BNgo6VFJ10i6Gvg5sHqwzCUtk7RFUrWkm/r4vFDSA+HnL0ianfDZzeH5LZIuGixPSfdIelnSK5IekpQ1T5Js7eii4UhHbALKwnCml4+jOJdZBmuhXBq+ioB9wPuBDwC1wISBLpSUC9wFXAwsAD4maUGvZNcCB81sHkEX2h3htQuAFcBCYBnwPUm5g+T5eTM71cxOAXYANw5ybxmjZ8pwXAJK2dgCqiaM8T29nMswg83y+tQx5H0GUG1mWwEkrQSWA5sS0iwHvhYePwT8o4J9zZcDK82sDdgmqTrMj/7yNLPG8JyAMfSxbiZTHV0lH5MxFAhWzHuXl3OZZbB1KABIKiJoTSwkaK0AYGZ/OsBllcDOhPc1wJn9pTGzTkkNwKTw/PO9rq0Mj/vNU9K/AJcQBK0v9nMv1wPXA8ycOXOA6sfH3pi1UCDYyv5XG/fS1NrB+KL8dFfHOZcCyQ7K3w9MJXiC47MET3BsGuSavhZE9G419JdmqOeDg6BFNR3YDFzVV6XM7G4zW2pmSysqKvpKEjv7YxhQehY4bvJWinMZI9mAMs/Mvgq0hPt7fRA4eZBraoAZCe+rgN39pZGUB5QCBwa4dtA8zawLeAD46KB3lSH2NbZSmJdDSVFSDc5RYdH0cMW8BxTnMkayAaUj/HpI0iKCX/yzB7lmNTBf0pxwlf0KYFWvNKuAq8PjK4CnzMzC8yvCWWBzgPnAi/3lqcA8ODqGcinwWpL3Fnu1TW1MLimM1WN1K8YXMqWk0Leydy6DJPsn7d2SJgBfJfhlXxwe9yscE7kReAzIBe41s42SbgXWmNkq4B7g/nDQ/QBBgCBM9yDBWEgncEPY8qCfPHOA+ySVEHSLvQz8r6S/CzFX29wWy517F00v9anDzmWQpAKKmf0wPHwWOC7ZzM3sUeDRXuduSThuBa7s59rbgNuSzLMbODvZemWa2qY2jiuP37KbhZWlPL1lP0fauxhTkJvu6jjnjlGye3lNkvQPkl6StFbStyVNirpyLjm1TW1UjI/PlOEei6aX0G2wea+PoziXCZIdQ1lJsNXKRwnGOuoIBr5dmrV3dnPwcEc8A0rPFiw+juJcRkh2DGWimf1NwvuvS7o8igq5oalvCRY1xjGgTCstYuK4At952LkMkWwL5WlJKyTlhK//AfxXlBVzyaltCgNKcfwCiiQWTi/xgXnnMsRgm0M2SWoE/ifwb0B7+FoJfD766rnBHA0oMWyhQNDt9fq+Jto6u9JdFefcMRowoJjZeDMrCb/mmFle+Moxs5KRqqTrX+wDyvRSOrqM1/YMtvGCc260S3pptaTLgPeFb58xs19EUyU3FD0BZVJxQZprMjxLZpYBsH7nIU6dUZbm2jjnjkWy04ZvBz5LsNBwE/DZ8JxLs9rmNsrG5lOYF891HNNKi5hSUsi6HQfTXRXn3DFKtoVyCbA4XECIpPuAdcA7HprlRlZtU1ssB+R7SGLJjAm8tONQuqvinDtGyc7yAkjsjyhNdUXc8MR1UWOi02aVsePAYeqa29JdFefcMUg2oHwDWCfpR2HrZC3w/6KrlktWbXP8A8qSmcHDP9d7K8W5WBs0oIS79/4WOAt4OHy928xWRlw3l4S4d3lBMNMrL0es2+njKM7F2aBjKGZmkh4xs3fxzu3nXRq1tHVyuL0r9i2UMQW5nDSthHXeQnEu1pLt8npe0umR1sQN
```

Comment:
My computer can do 10000 convolutions in about 10 seconds. I'm not sure you are limited to 10.